lib/Ebizmarts/MailChimp.php

```diff
@@ -417,4 +423,37 @@ public function call($url, $params, $method = Ebizmarts_MailChimp::GET, $encodeJ
 
         return $result;
     }
+
+    /**
+     * @return array
+     */
+    public function getCurlOptions()
+    {
+        return $this->_curlOptions;
+    }
+
+    /**
+     * @param array $_curlOptions
+     */
+    protected function setCurlOptions($curlOptions)
+    {
+        $this->_curlOptions = $curlOptions;
+    }
+
+    /**
+     * @param $index
+     * @param $value
+     */
+    protected function setCurlOptionsIndexValue($index, $value)
+    {
+        $this->_curlOptions[$index] = $value;
+    }
+
+    /**
+     * @param array $curlOptions
+     */
+    protected function setCurlOptionsAddOptions($curlOptions = array())
+    {
+        $this->_curlOptions += $curlOptions;
+    }
```

Comment:
What is the purpose of these functions?

Reply:
They was intended for being used if needed. Now they are removed.